cmd/agent.go

```diff
@@ -41,7 +41,7 @@ func init() {
 		&agent.Period,
 		"period",
 		"p",
-		time.Hour,
+		0,
 		"Time between scans (given as XhYmZs).",
```

Comment:
```suggestion
		"Override time between scans in configuration file (given as XhYmZs).",
```

Reply:
Fixed in 0b00c186f8cdc4753036be85be1993de6c586e8f